Assets/Scripts/Overworld/CharacterOW.cs

```diff
@@ -26,67 +26,111 @@ public class CharacterOW : MonoBehaviour, IOverworldSkillListener {
             get => Party.Instance.characters?[0]?.OverworldInstance?.CanMove ?? false;
             set {
                 foreach (Character character in Party.Instance.characters) {
-                    if (character != null && character.OverworldInstance) {
-                        character.OverworldInstance.CanMove = value;
-                    }
+                    SetCharacterInstanceCanMove(character, value);
                 }
             }
         }
 
+        private static void SetCharacterInstanceCanMove(Character character, bool value) {
+            if (character != null && character.OverworldInstance) {
```

Comment:
```suggestion
            if(character?.OverworldInstance == null)
                return;
            character.OverworldInstance.CanMove = value;
```
PS: character.OverworldInstance is an object, right? you should check for null.